packages/shared/friends/sagas.ts

```diff
@@ -360,15 +365,42 @@ function* refreshFriends() {
   }
 }
 
-// function* getFriends(request: GetFriendsPayload) {
-//   // ensure friend profiles are sent to renderer
+export function getFriends(request: GetFriendsPayload) {
```

Comment:
I'm wondering about keeping this as a generator instead, but not at all a blocker here. Is there a reasoning to prefer it this way?